api/auth/oauth_handlers.py

```
@@ -42,7 +42,7 @@ async def handle_callback(provider: str, user_info: Dict[str, Any], api_token: s
             )
 
             return True
-        except Exception as exc:  # capture exception for logging
+        except Exception as exc:  # capture exception for logging, pylint: disable=broad-exception-caught
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)